aaa_base.spec

```diff
@@ -107,6 +108,7 @@ systems.
 Summary:        sysctl setting to allow ptrace with the YAMA LSM enabled
 Group:          System/Fhs
 Requires:       %{name} = %{version}
+Requires:       procps
```

Comment:
no, that would be a `Requires(post)`. 

but this is super annoying. can we just echo 0 > .. ?

Reply:
Please, no more dependencies to aaa_base, all of this dependencies will be pulled in our containers, images, etc, even if they are useless there and not needed.

Reply:
okay, removed it